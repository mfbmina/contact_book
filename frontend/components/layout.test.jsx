import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import Layout from './layout';

describe('Layout', () => {
  it('should render', () => {
    const component = shallow(<Layout title="test"><p>Testing</p></Layout>);

    expect(component).toMatchSnapshot();
  });
});
