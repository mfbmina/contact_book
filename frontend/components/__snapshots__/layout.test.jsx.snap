// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render 1`] = `ShallowWrapper {}`;
